src/shared/tpm2-util.h

```diff
@@ -11,6 +11,18 @@ typedef enum TPM2Flags {
         TPM2_FLAGS_USE_PIN = 1 << 0,
 } TPM2Flags;
 
+
+/* As per https://trustedcomputinggroup.org/wp-content/uploads/TCG_PCClient_PFP_r1p05_v23_pub.pdf a
+ * TPM2 on a Client PC must have at least 24 PCRs. This hardcodes our expectation of 24. */
+#define TPM2_PCRS_MAX 24U
+#define TPM2_PCRS_MASK ((UINT64_C(1) << TPM2_PCRS_MAX) - 1)
+static inline bool TPM2_PCR_VALID(uint64_t pcr) {
+        return pcr < TPM2_PCRS_MAX;
+}
+static inline bool TPM2_PCR_MASK_VALID(uint64_t pcr_mask) {
```

Comment:
similar here, so far we used uint32_t for pcr masks afaics

Reply:
this was actually already uint64_t since df6c3cbd02b, but i agree we only need 32, will change it